cli/src/factory_ops/unboxing/init

```diff
@@ -329,41 +329,19 @@ provisioner pillar_set commons/repo/lustre \"\" --logfile --logfile-filename ${L
 echo "Removing the repos with Seagate URL from Server A" | tee -a ${LOG_FILE}
 seagate_refs_cleanup
 
-echo "Restarting the cluster resources.........." | tee -a ${LOG_FILE}
-echo -n "ClusterIP.............................." | tee -a ${LOG_FILE}
-pcs resource refresh ClusterIP:0 >> ${LOG_FILE}
-pcs resource refresh ClusterIP:1 >> ${LOG_FILE}
-sleep 5
-echo "Ok." | tee -a ${LOG_FILE}
-
-echo -n "Stonith................................" | tee -a ${LOG_FILE}
-pcs resource refresh stonith-c1 >> ${LOG_FILE}
-pcs resource refresh stonith-c2 >> ${LOG_FILE}
-echo "Ok." | tee -a ${LOG_FILE}
-sleep 5
-
-echo -n "CSM & Kibana..........................." | tee -a ${LOG_FILE}
-pcs resource refresh csm-kibana >> ${LOG_FILE}
-sleep 5
-echo "Ok." | tee -a ${LOG_FILE}
-
-echo -n "UDS...................................." | tee -a ${LOG_FILE}
-pcs resource refresh uds >> ${LOG_FILE}
-sleep 5
-echo "Ok." | tee -a ${LOG_FILE}
-
-echo -n "HAProxy................................" | tee -a ${LOG_FILE}
-pcs resource refresh haproxy-c1 >> ${LOG_FILE}
-pcs resource refresh haproxy-c2 >> ${LOG_FILE}
-echo "Ok." | tee -a ${LOG_FILE}
+echo "Restarting the Cortx Services" | tee -a ${LOG_FILE}
```

Comment:
Did we not changed this message - "Restarting the cluster resources through switching maintenance mode"

Reply:
Yes, I see the change not pushed to pre-cortx-1.0 as well